### PR TITLE
Add cli option to update workflow

### PR DESCRIPTION
**Pull request by katrogan**
# TL;DR
This change allows a user to update workflow state using the flyte-cli

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [x] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Tested on flyte staging and checked on portal that the state was correctly modified.

## Tracking Issue
https://github.com/lyft/flyte/issues/235

## Follow-up issue
_NA_

### CODE REVIEW

File: flytekit/clis/flyte_cli/main.py [hunk 1]
```diff
@@ -403,6 +403,12 @@ def _render_schedule_expr(lp):
     required=True,
     help="Whether or not to set schedule as active."
 )
+_workflow_state_choice = _click.option(
+    "--state",
+    type=_click.Choice(["active", "archived"]),
+    required=True,
+    help="Whether or not to set a workflow as actively visible in the UI."
```
Comment (wild-endeavor) [inline flytekit/clis/flyte_cli/main.py:410]: actively visible?

File: flytekit/clients/friendly.py [hunk 2]
```diff
@@ -317,6 +318,25 @@ def get_workflow(self, id):
             )
         )
 
+    def update_workflow(self, id, state):
+        """
+        This updates the state of the workflow specified by the ID.
+        :param id: flytekit.models.core.identifier.Identifier id: The ID representing a given task.
+        :param int state: Enum value from flytekit.models.workflow.WorkflowState
+        :return:
```
Comment (wild-endeavor) [inline flytekit/clients/friendly.py:326]: if this passes the return type from raw, can you copy paste here too?

File: flytekit/clis/flyte_cli/main.py (hunk 1 above)
Comment (katrogan) [reply flytekit/clis/flyte_cli/main.py:410]: trying to tie it back to 'active'... i'll reword

Comment (kumare3) [reply flytekit/clis/flyte_cli/main.py]: We should say Ui/ cli. Default is active only

**Pull request merged**